CVLByExample/UnresolvedCallSummarization/README.md

```diff
@@ -0,0 +1,31 @@
+# Unresolved Call Summarization Example
+
+## Overview
+This repository contains a smart contract named `TrusterLenderPool`, featuring a `flashLoan` function. However, there exists a significant vulnerability within this function. The vulnerability allows the pool contract to act as the `msg.sender` for any call, presenting a critical security flaw. This example showcases the utilization of a new feature in CVL called Unresolved Call Summarization. This feature accentuates how the vulnerability can be exploited to grant the attacker an infinite allowance, subsequently leading to the potential draining of all funds within the contract.
```

Comment:
add a note that this is based on damnVulnerability and give a link

Reply:
also can we estimate what will be the link in the documentation, this is the PR:
https://github.com/Certora/Documentation/pull/236

Reply:
Done